### PR TITLE
[Spritelab] makeNumSprites block

### PR DESCRIPTION
**Pull request by ajpal**
Another low-lift, ceiling-raising block for Sprite Lab.
image
Makes the specified number of sprites at random locations
image

### CODE REVIEW

File: dashboard/config/blocks/GamelabJr/gamelab_makeNumSprites.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+        "type": "Number",
+        "field": true
+      },
+      {
+        "name": "ANIMATION_NAME",
+        "customInput": "costumePicker"
+      }
+    ]
+  }
+}
```
Comment (epeach) [inline dashboard/config/blocks/GamelabJr/gamelab_makeNumSprites.json:24]: Does the linter need a new line at the end of this file?

**Review by epeach — APPROVED** [review]
LGTM

File: dashboard/config/blocks/GamelabJr/gamelab_makeNumSprites.json (hunk 1 above)
Comment (ajpal) [reply dashboard/config/blocks/GamelabJr/gamelab_makeNumSprites.json:24]: No, all of the block config files are auto-generated through the levelbuilder UI and don't have the extra line at the end.

Comment (epeach) [reply dashboard/config/blocks/GamelabJr/gamelab_makeNumSprites.json:24]: TIL! Thanks!

**Pull request merged**